src/main/java/mate/hwdao/dao/ManufacturerDao.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Optional;
 import mate.hwdao.model.Manufacturer;
 
-public interface ManufacturerDao {
+public interface ManufacturerDao extends GenericDao<Manufacturer, Long> {
     Manufacturer create(Manufacturer manufacturer);
 
     Optional<Manufacturer> get(Long id);
```

Comment:
No need to keep the same abstract methods in ManufacturerDao, as they dublicate GenericDao methods. Feel free to remove them